src/data/data.js

```diff
@@ -0,0 +1,16 @@
+const words = [
+    {
+        value: 'abrazje',
+        anagrams: ['bajarze'],
```

Comment:
Agree to changing data structure (all possible words in one array - it will simplify checking the game's result). However I still think data should be array of objects, where we have one main word which is the word drawn by the app at the very beginning. Otherwise, how choosing this word would look like. Possibility of drawing anagrams should be the same as the possibility of drawing the main word. 

In other words: let's imagine we have an array of 10 items (arrays or objects) in total. Possibility of drawing 1 of these 3 words (`'kroniki', 'korniki', 'krionik'`) by the app should be 3:10. And possibility of drawing each of these words should be equal to possibility of drawing every other word (with anagrams or not - this should not determine the possibility). So each word should have it's own object.
Plus there should be only one draw in the game, not two (in case of array of arrays - first index and the second one).

Additionally I feel (although I'm not a database guy) such a structure it's easier to create and maintain.

Reply:
I think of it this way: before a game starts we want to draw 7-element set of letters which can happen to form one or more words. Considering game rules there is no such thing as `main word` and its `anagrams`.

I didn't consider probabilities of drawing one-word letter sets vs multi-word letter sets. I thought every set of letters will have the same probability. But if your design decision is to make multi-word letter sets more probable than let's do it :)

So initially proposed data structure was:
```
[
{
  value: 'korniki',
  anagrams: ['kroniki', 'krionik'],
},
]
```

If I understand correctly, now the data structure you think about looks this way ("array of objects"):
```
[
{
  word: 'kroniki',
  anagrams: ['kroniki', 'korniki', 'krionik'],
},
{
  word: 'korniki',
  anagrams: ['kroniki', 'korniki', 'krionik'],
},
{
  word: 'krionik',
  anagrams: ['kroniki', 'korniki', 'krionik'],
},
]
```
My concern is this data structure is redundant, it makes `data` bigger than needed. For example string value `'kroniki'` is specified four times (and some letter sets have 12 words, see http://www.pfs.org.pl/slowneroznosci.php). I expect `data` to be huge, I wonder if it's too huge loaded to be loaded to memory so making it even bigger may cause problems. Except this concern this data structure looks much better to me as it's symmetrical, it does not favour `korniki`.

The alternative would be much more dense:
```
[
  ['kroniki', 'korniki', 'krionik'],
]
```
or in unpacked form that's ready to be drawn from (this can be derived from the above with a simple `reduce`):
```
[
  ['kroniki', 'korniki', 'krionik'],
  ['kroniki', 'korniki', 'krionik'],
  ['kroniki', 'korniki', 'krionik'],
]
```
Okay, I think I've made my point. Please feel free to decide what you like the most and let's go on.

Reply:
Okay, let's do an array of arrays